azure/scope/clients.go

```diff
@@ -25,13 +25,11 @@ import (
 	"strings"
 
 	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
-	azureautorest "github.com/Azure/go-autorest/autorest/azure"
-	"github.com/Azure/go-autorest/autorest/azure/auth"
 )
 
 // AzureClients contains all the Azure clients used by the scopes.
 type AzureClients struct {
-	auth.EnvironmentSettings
+	EnvironmentSettings
```

Comment:
I'm a little confused about how we were really using this type before if not just to store data. Isn't this a SDKv1 construct, and presumably not usable with SDKv2? Should we be utilizing this instead? https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/azcore@v1.11.1/cloud

Reply:
I am also confused; AFACIT we've just been using this as an environment variable container and I've tried to keep the usages intact so as not to break things, but it may all amount to a no-op honestly.

Let me look more closely at `Configuration` and see if it's a more functional replacement for this go-autorest code path. Thanks @nojnhuh!